src/Voron/Util/ActiveTransactions.cs

```diff
@@ -30,6 +30,22 @@ public void Add(LowLevelTransaction tx)
             }
         }
 
+        public void RecheckOldestTransaction()
```

Comment:
I wonder if we can have a more specific name for this guy.  Maybe `ForceRecheckingOldestTransactionByFlusherThread` ?
I'd like to have something more explicit here.

Reply:
Fine by me